developers_affiliations6.txt

```diff
@@ -24041,6 +24041,3 @@ seanmcguigan: seanmcguigan!users.noreply.github.com
 	agora from 2020-10-01 until 2021-03-01
 	Formidable Labs LLC from 2021-03-01 until 2022-07-01
 	Sage from 2022-07-01
-seanmonstar: sean!buoyant.io, sean!seanmonstar.com, sean.monstar!gmail.com
```

Comment:
Note that this user was at the bottom of file #6 and at the top of file #7 with different data.  I've updated the correct data in #7.

Reply:
Noted, I will check this and eventually also fix post-merge.